cpp/velox/compute/VeloxBackend.cc

```diff
@@ -80,13 +80,14 @@ void VeloxInitializer::Init(std::unordered_map<std::string, std::string> conf) {
   filesystems::registerHdfsFileSystem();
   // TODO(yuan): should read hdfs client conf from hdfs-client.xml from
   // LIBHDFS3_CONF
-  std::string hdfsUri = "localhost:9000";
+  std::string hdfsUri = conf["spark.fs.defaultFS"];
```

Comment:
does this mean gluten can not read from other HDFS endpoint, only "defaultFS" can work?

Reply:
Thanks your comment.
Gluten could still read from other HDFS endpoint by set env variable `VELOX_HDFS` explicitly, this pr does not change this behavior, it just add a new way to read from `core-site.xml` if we are in fully-prepared hadoop cluster.

btw I'm confused about why use env variable to pass conf, gluten already support pass conf from spark to native backend.